CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ endif()
 # Versions MUST contain three parts and start with lowercase 'v'.
 # Example 'v1.0.0'. They MUST not contain a pre-release label such as '-beta'.
 set(FIRMWARE_VERSION "v4.0.0")
+set(FIRMWARE_BTC_VERSION "v6.0.0")
```

Comment:
Should be v4.0.0, we can diverge later, but for the initial version should be the same so all previous version checks work.

Reply:
oh yeah, I was just testing, needed it to be different than `4.0.0`. I forgot to set it to 4 again.